.circleci/config.yml

```diff
@@ -29,12 +29,11 @@ jobs:
       - image: astronomerio/ap-build:0.1.3
     steps:
       - checkout
-      - run:
-          name: Install ci tools
-          command: ./bin/install-ci-tools 1
       - run:
           name: Package the Astronomer chart
-          command: make build
+          command: |
+            source ./bin/install-ci-tools 1
+            make build
```

Comment:
This change is what fixes the issue. previously, "helm" in path for the build was Helm 2